dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -30,6 +34,24 @@ get_define() {
     printf "%s" "$line"
 }
 
+get_kernel_info() {
+    printf "version=%s machine=%s processor=%s" $(uname -r) $(uname -m) $(uname -p)
```

Comment:
Why not just `uname -a`?

Reply:
Also, the uname calls must be quoted, I get this weird output on my machine:

```
Kernel: version=4.18.7-gentoo machine=x86_64 processor=Intel(R)version=Core(TM) machine=i7-6820HK processor=CPUversion=@ machine=2.70GHz processor=
```

Reply:
it now prints output of `uname -mprs` directly, `uname -a` includes the hostname which is not needed or might also compromise security/privacy if someone posts this output from their system in public.